CHANGELOG.md

```diff
@@ -22,8 +22,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
- - **CUMULUS-2124**
-   - cumulus-rds-tf terraform module now takes engine_version as an input variable.
+- **CUMULUS-2279**
```

Comment:
Nit: would like to keep the items in ticket number ascending order, though I can see that nothing in here is really following that guideline

Reply:
https://github.com/nasa/cumulus/pull/2014/commits/9add0d151155da2f69fee5ead7a6acbbb9233c37